CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+* The `role_revoke` function now takes a `%Role{}`, and a `%Group{}` instead of
+  names. It returns a `%Group{}` with the associated roles as an attribute. The
+  original `role_revoke` can still be used for users.
```

Comment:
I have been leaving functionality in `Old` for functions that are querying by name. I would imagine that CogCtl will want to continue to query by name, so I figured it would be better to leave it in until somebody decides to port over that functionality.